nglview/show.py

```diff
@@ -281,6 +282,25 @@ def show_schrodinger_structure(mol, **kwargs):
     return NGLWidget(structure_trajectory, **kwargs)
 
 
+def show_biopython(entity, **kwargs):
+    '''Show NGL widget with Biopython structural entity.
+
+    Takes a Structure, Model, Chain, Residue or Atom
+    from Bio.PDB as its data input.
+
+    Examples
+    --------
+    >>> import nglview as nv # doctest: +SKIP
+    ... from Bio.PDB import PDBParser
+    ... parser = PDBParser()
+    ... structure = parser.get_structure("protein", "protein.pdb")
+    ... w = nv.show_biopython(structure[0]["A"])
+    ... w
+    '''
+    entity = BiopythonStructure(entity)
+    return show_text(entity.get_structure_string(), **kwargs)
```

Comment:
I think this should be changed to

```python
return NGLWidget(entity)
```

Check other classes for reference.
The point of making that interface is here

```
view = NGLWidget()
view.add_structure(entity)
ase_structure = ASEStructure(...)
view.add_structure(ase_structure)
```

cool, huh? :D